docs/source/gateway/index.rst

```diff
@@ -244,61 +244,18 @@ The table below presents a non-exhaustive list of models and a corresponding rou
 With the rapid development of LLMs, there is no guarantee that this list will be up to date at all times. However, the associations listed
 below can be used as a helpful guide when configuring a given route for any newly released model types as they become available with a given provider.
 
```

Comment:
Can we add some text that lets a reader know that the `N/A` values actually mean that the provider currently doesn't support this interface? 
At a quick cursory glance at the table it's hard to know if the `N/A` is a placeholder that we just didn't fill in or that it's actually an unavailable route configuration.

Reply:
Updated!